specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -522,16 +522,22 @@
           "description": "The status of the email",
           "type": "string",
           "enum": [
-            "Failed",
+            "Bounced",
             "Delivered",
+            "Failed",
             "FilteredSpam",
-            "Quarantined"
+            "Quarantined",
+            "Suppressed"
           ],
           "x-ms-enum": {
             "name": "AcsEmailDeliveryReportStatus",
             "modelAsString": true
           }
         },
+        "deliveryStatusDetails": {
```

Comment:
We need an example that shows this.

Reply:
Potentially make this an object so it can evolve over time, and the string information can go in a `Message` or `Description` property.

Reply:
Updated this property to be an object with 'statusDescription' as one of the properties.